### PR TITLE
Change correlationID to sdkCorrelationID

### PR DESCRIPTION
**Pull request by nbierdeman**
### Purpose

This PR is step 2 of 3 in the ["Rename prop to sdkCorrelationID"](https://github.com/paypal/paypal-checkout-components/pull/1585) PR of **paypal-checkout-components**. It replaces the `correlationID` prop with the `sdkCorrelationID` prop.

### CODE REVIEW

File: src/props/onInit.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import { ZalgoPromise } from 'zalgo-promise/src';
 
 export type XOnInitDataType = {|
-    correlationID : string
+    sdkCorrelationID : string
```
Comment (bluepnume) [inline src/props/onInit.js:6]: We can't change this key, this is a public interface

File: src/props/onInit.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ export function buildXOnInitActions(set : (boolean) => void) : XOnInitActionsTyp
 }
 
 export type OnInitDataType = {|
-    correlationID : string
+    sdkCorrelationID : string
```
Comment (bluepnume) [inline src/props/onInit.js:24]: Same.

File: src/button/button.js [hunk 3]
```diff
@@ -75,7 +75,7 @@ export function setupButton(opts : ButtonOpts) : ZalgoPromise<void> {
     
     const components = getComponents();
 
-    const { initPromise, isEnabled } = onInit({ correlationID: buttonCorrelationID });
+    const { initPromise, isEnabled } = onInit({ sdkCorrelationID: buttonCorrelationID });
```
Comment (bluepnume) [inline src/button/button.js:78]: Recommend leaving this. `buttonCorrelationID !== sdkCorrelationID`

File: src/button/props.js [hunk 4]
```diff
@@ -51,8 +51,7 @@ export type ButtonXProps = {|
     buttonSessionID : string,
     clientID : string,
     partnerAttributionID : ?string,
-    correlationID : string,
-    sdkCorrelationID? : string,
+    sdkCorrelationID : string,
```
Comment (bluepnume) [inline src/button/props.js:54]: This looks good

File: src/button/props.js [hunk 5]
```diff
@@ -189,8 +188,7 @@ export function getProps({ facilitatorAccessToken } : {| facilitatorAccessToken
         clientID,
         partnerAttributionID,
         clientMetadataID,
-        correlationID,
-        sdkCorrelationID = correlationID,
+        sdkCorrelationID,
```
Comment (bluepnume) [inline src/button/props.js:191]: This looks good

File: src/button/button.js [hunk 6]
```diff
@@ -33,7 +33,7 @@ type ButtonOpts = {|
     eligibility : {|
         cardFields : boolean
     |},
-    correlationID? : string,
+    sdkCorrelationID? : string,
```
Comment (bluepnume) [inline src/button/button.js:36]: Please leave this. this is button corrid not sdk corrid.

File: src/button/button.js [hunk 7]
```diff
@@ -56,7 +56,7 @@ export function setupButton(opts : ButtonOpts) : ZalgoPromise<void> {
     }
 
     const { facilitatorAccessToken, eligibility, fundingEligibility, buyerCountry: buyerGeoCountry, sdkMeta, buyerAccessToken, wallet, cookies,
-        cspNonce: serverCSPNonce, merchantID: serverMerchantID, firebaseConfig, content, personalization, correlationID: buttonCorrelationID = '' } = opts;
+        cspNonce: serverCSPNonce, merchantID: serverMerchantID, firebaseConfig, content, personalization, sdkCorrelationID: buttonCorrelationID = '' } = opts;
```
Comment (bluepnume) [inline src/button/button.js:59]: Same.

**Comment by bluepnume** [comment]
Please remove all changes except those in props.js :) button session id is a different id, we should not be changing.

**Comment by nbierdeman** [comment]
Thanks for the feedback, @bluepnume! Will start working on this now.

**Comment by nbierdeman** [comment]
@bluepnume @gregjopa I reverted some of my original changes based on Daniel's feedback, and am ready for another review. Thanks!

**Review by gregjopa — APPROVED** [review]
Looks good to me!

**Pull request merged**